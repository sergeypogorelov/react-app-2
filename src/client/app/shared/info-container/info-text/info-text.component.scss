@import '../../../../styles/variables';

.info-text {
  margin-bottom: 0;
  font-family: $f-extraBold;
  font-size: 16px;
}
