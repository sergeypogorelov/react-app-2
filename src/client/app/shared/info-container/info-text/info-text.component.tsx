import './info-text.component.scss';

import React, { FunctionComponent } from 'react';

export const InfoText: FunctionComponent<{}> = ({ children }) => {
  return <p className="info-text">{children}</p>;
};
