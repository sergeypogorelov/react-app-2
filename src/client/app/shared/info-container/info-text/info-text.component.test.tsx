import React from 'react';
import renderer from 'react-test-renderer';

import { InfoText } from './info-text.component';

test('InfoText component should render correctly', () => {
  const tree = renderer.create(<InfoText>test</InfoText>).toJSON();
  expect(tree).toMatchSnapshot();
});
