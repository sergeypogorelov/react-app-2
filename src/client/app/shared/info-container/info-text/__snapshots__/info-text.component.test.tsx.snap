// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoText component should render correctly 1`] = `
<p
  className="info-text"
>
  test
</p>
`;
